index.d.ts

```diff
@@ -1317,7 +1317,112 @@ export interface AWS {
           [k: string]: unknown;
         };
   };
+  stepFunctions?: {
```

Comment:
See https://github.com/serverless/typescript#contributing
All modifications made to `index.d.ts` will be overwritten at each release of this package. No modification shall be done manually to this file.